doc/releases/release-notes-3.0.rst

```diff
@@ -278,6 +278,7 @@ Networking
 
 * Misc:
 
+  * gptp: clock sync ratio as double, not float
```

Comment:
If you want this PR to be part of the 3.0 release, you need to add a milestone and bring it to the attention of the release manager.

Reply:
Thanks.  I would suggest changing this entry to `grtp: convert clock sync ratio from float to double` as to me the current text is confusing.

Reply:
@dkalowsk, I was specially recommending to @ainguraXmarquiegui, that he separate the release notes change to a separate PR that is pending based on whether this PR gets merged.

Reply:
Thank you @dleach02 . I have never used the Milestone feature on github.

If I create a milestone in my forked repo, I can't assign this pull request to that milestone, and I don't have permissions to create milestones here.

How does this typically work?

Should I remove the release notes commit from this PR and move it to a new PR so that someone with permissions can create a new milestone on this repo?

Or should I create the milestone along with two new PR on my own fork (if that's even possible)?

Reply:
the milestone selection is for this particular PR. It just needs the 3.0 selected. I can set the milestone if that is what you want.

I do want you to separate out the release note changes into a standalone PR that is pending this one being merged.

Reply:
OK, I removed the commit from here and created https://github.com/zephyrproject-rtos/zephyr/pull/42624 with only the release notes commit.

Let me know if this is good enough.